sdk/storage/azure-storage-queue/azure/storage/queue/_parser.py

```diff
@@ -0,0 +1,17 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+DEFAULT_OAUTH_SCOPE = "/.default"
```

Comment:
Do we even need this constant or should I just slap in `/.default` naked? And then IF we are going to keep the constant, I wanted it to match the other definition of `DEFAULT_OAUTH_SCOPE` elsewhere so it has to include to `/` in it.

Reply:
Probably don't need the constant. It's also already defined in constants.